packages/brand/src/Brand/Brand.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { assetUrl } from "@kaizen/hosted-assets"
+
+export interface BrandProps {
+  variant:
+    | "logo-horizontal"
+    | "logo-vertical"
+    | "enso"
+    | "collective-intelligence"
+  isReversed: boolean
```

Comment:
Soz I missed this on the previous look overs. Can this prop name please be `reversed`? Looking over other components with a reversed prop, there are two components with `isReversed` but the vast majority have this prop named `reversed`. It can also be optional.

Reply:
@dougmacknz no worries. This has been updated now 👍